Interface/AddOns/oUF_Neav/config.lua

```diff
@@ -135,9 +135,16 @@ ns.Config = {
 
             numBuffs = 20,
             numDebuffs = 20,
+
+            disableAura = false,                                                            -- Disable Auras on this unitframe
             colorPlayerDebuffsOnly = true,
             showAllTimers = false,                                                          -- If false, only the player debuffs have timer
-            disableAura = false,                                                            -- Disable Auras on this unitframe
+            onlyShowPlayer = false,                                                          -- Only shows player created buffs/debuffs.
+
+            -- Debuffs on top options. Overrides onlyShowPlayer option.
+            showDebuffsOnTop = false,
+            onlyShowPlayerDebuffs = true,
+            onlyShowPlayerBuffs = false,
```

Comment:
I'm not sure I'm convinced it's a good idea to have separate options for when the debuffs are on top, or below. It might just end up being confusing for most people. What do you think?